src/dwarf.rs

```diff
@@ -891,10 +913,23 @@ impl DwarfResolver {
         }
         addr_to_dlcu.sort_by_key(|v| v.0);
 
+        let debug_info_syms = if debug_info_symbols {
```

Comment:
I have another patch set for this purpose for symbolizing addresses and finding the addresses of symbols.  It will introduce a new option to enable or disable fully loading in advance.

Reply:
great, what about lazy loading? is that going to be supported?

Reply:
yes!